test/common_tester.h

```diff
@@ -8,9 +8,7 @@
 #define MANIF_TEST(manifold)                                              \
```

Comment:
Did you run all the standard tests on Rn?

Reply:
Yes, the tests do pass locally and on Travis CI. See the tests [here](https://github.com/artivis/manif/blob/feature/rn/test/rn/gtest_rn.cpp#L15) and Travis CI [here](https://travis-ci.com/artivis/manif/builds/136435353).

Reply:
Does your answer still apply after further commits?

Reply:
Yes, only the documentation-related test is failing. I'll look it up.